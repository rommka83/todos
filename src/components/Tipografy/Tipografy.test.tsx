import React from 'react';
import renderer from 'react-test-renderer';
import { Tipografy } from './index';

describe('проверка Tipografy', () => {
  test('отрисовка Tipografy, вариант - "text-main"', () => {
    const component = renderer.create(<Tipografy variant={'text-main'}>Тест</Tipografy>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('отрисовка Tipografy, вариант - "text-body"', () => {
    const component = renderer.create(<Tipografy variant={'text-body'}>Тест</Tipografy>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('отрисовка Tipografy, вариант - "text-completed"', () => {
    const component = renderer.create(<Tipografy variant={'text-completed'}>Тест</Tipografy>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('отрисовка Tipografy, вариант - "text-supporting"', () => {
    const component = renderer.create(<Tipografy variant={'text-supporting'}>Тест</Tipografy>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
