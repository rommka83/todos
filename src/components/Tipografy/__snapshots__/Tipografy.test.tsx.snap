// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`проверка Tipografy отрисовка Tipografy, вариант - "text-body" 1`] = `
<p
  className="root textBody"
>
  Тест
</p>
`;

exports[`проверка Tipografy отрисовка Tipografy, вариант - "text-completed" 1`] = `
<p
  className="root textCompleted"
>
  Тест
</p>
`;

exports[`проверка Tipografy отрисовка Tipografy, вариант - "text-main" 1`] = `
<p
  className="root textMain"
>
  Тест
</p>
`;

exports[`проверка Tipografy отрисовка Tipografy, вариант - "text-supporting" 1`] = `
<p
  className="root textSupporting"
>
  Тест
</p>
`;
